core/scss/components/lockup/_lockup.scss

```diff
@@ -0,0 +1,148 @@
+@charset "UTF-8";
+
+//
+// Lockup
+//
+// A lockup with the Stanford wordmark logo and department/unit name. It is used in the header of a website and links to the homepage of the department/unit. When building a department lockup, besides the Stanford wordmark, the most obvious element should be the most granular relevant department.
+//
+// Markup: ../templates/components/lockup/lockup.twig
+//
+// Style guide: Components.Lockup
+//
+.su-lockup {
+  @include display-grid;
+  @include grid-columns(1fr);
+  @include grid-rows(auto auto auto auto);
+  align-items: end;
+  width: fit-content;
+  text-decoration: none;
+
+  @media #{$breakpoint-md} {
+    @include grid-columns(auto auto 1fr);
+    @include grid-rows(auto auto);
+  }
+}
+
+.su-lockup__wordmark {
+  @include logo;
+  display: block;
+  -ms-grid-row: 1;
+  -ms-grid-row-span: 1;
+  grid-row: 1 / span 1;
+  -ms-grid-column: 1;
+  -ms-grid-column-span: 1;
+  grid-column: 1 / span 1;
+  color: $color-cardinal-red;
+  font-size: 36px;
+  line-height: 1;
+
+  @media #{$breakpoint-md} {
+    font-size: 46px;
+    height: 33px;
+    line-height: .96;
+  }
+
+  // Hide pipe/horizontal line on smaller devices if there is no line 5
+  + .su-lockup__pipe {
+    display: none;
+
+    @media #{$breakpoint-md} {
+      display: block;
+    }
+  }
+}
+
+//Put the pipe in its own CSS grid cell so that its height auto adjusts to the tallest part of the lockup
+.su-lockup__pipe {
+  @include margin(7px null);
+  display: block;
+  -ms-grid-row: 3;
+  -ms-grid-row-span: 1;
+  grid-row: 3 / span 1;
+  -ms-grid-column: 1;
+  -ms-grid-column-span: 1;
+  grid-column: 1 / span 1;
+  content: "";
+  width: 118px;
+  height: 1px;
+  background-color: $color-black;
+
+  @media #{$breakpoint-md} {
+    @include margin(0 7px);
+    grid-row: 1 / span 1;
+    -ms-grid-row: 1;
+    -ms-grid-row-span: 1;
+    grid-column: 2 / span 1;
+    -ms-grid-column: 2;
+    -ms-grid-column-span: 1;
+    display: inline-block;
+    width: 1px;
+    height: 100%;
+  }
+
+}
+
+.su-lockup__sitename-line1,
+.su-lockup__sitename-line2,
+.su-lockup__sitename-line5 {
+  @include font-smoothing;
+  display: block;
+  grid-column: 1 / span 1;
+  -ms-grid-column: 1;
+  -ms-grid-column-span: 1;
+  color: $color-black;
+  line-height: .8;
```

Comment:
Actually, same comment applies to line 1. Try "FREEMAN SPOGLI INSTITUTE FOR INTERNATIONAL STUDIES" for line 1.

Probably the same for the other lines???

Reply:
I made some tweaks so all lines can now wrap. I did refactor the line height/padding of line 5, but I'd rather leave the other lines alone since the pipe height kind of depends on the other line heights. Perhaps that would encourage people to not have super long site names that wrap to the 2nd line? 😄

Reply:
And maybe they could use the 2 line variant if the site name needs 2 lines. They can have both default for line 1 and line 2 and it would look like it's a single line wrapped to the 2nd.